resources/type-system/inference.md

```diff
@@ -220,15 +224,29 @@ return type of the context function type is used at several points during
 inference.  We refer to this type as the **imposed return type
 schema**. Inference for each returned or yielded expression in the body of the
 function literal is done using a context type derived from the imposed return
-type schema as follows:
+type schema `S` as follows:
```

Comment:
It looked like we had two concepts: 'imposed return type schema', and 'imposed return type'. The former was defined explicitly, and the context type relied on the latter. So I introduced a name for the imposed return type schema to avoid too many repetitions of that long phrase, and adjusted the text to use that name, which makes it obvious that there is no separate notion of an 'imposed return type'.